run_dir/static/js/project_samples.js

```diff
@@ -1179,7 +1179,7 @@ function load_charon_summary(){
           $('#charon-status-failed').text(data['failed']);
           $('#charon-status-runn').text(data['runn']);
           $('#charon-status-hge').text(data['hge']);
-          $('#charon-status-gdp').text(data['gdp'].toString().replace(/\'/," "));
+          $('#charon-status-gdp').text(data['gdp'].toString().replace(/\'/," ").replace(/,/g, ", "));
```

Comment:
I think it'll look better with multiple badges than putting everything in one huge badge.

Reply:
I agree! In that case I think it looks better without commas. I'll put it on stage...